src/components/MovieItem/MovieItem.jsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
 import styles from './MovieItem.scss';
 
 const cx = classNames.bind(styles);
 
-const MovieItem = ({ title, imageUrl }) => (
+const MovieItem = ({ title, imageUrl, movieObj }) => (
```

Comment:
If you pass the whole movie than you don't need to keep title and imageUrl props separately. You can take them from movieObj